### ISSUE TITLE
Limit number of provisioned devices per org

### ISSUE THREAD

**Issue by mobileoverlord** [nerves-hub/nerves_hub_web#217]
We should limit this to 5 devices per org. The setting should be configurable. As part of implementing this, we need to allow devices to be deleted.

**Comment by jlmcgehee21** [comment]
@mobileoverlord, would you want this config to be at the umbrella level or at the `nerves_hub_core` level?

**Comment by mobileoverlord** [comment]
Yeah we should probably handle it there for the API's sake since it does not have www as a dependency
